multichain-testing/test/stake-ica.test.ts

```diff
@@ -0,0 +1,220 @@
+import anyTest from '@endo/ses-ava/prepare-endo.js';
+import type { TestFn } from 'ava';
+import { commonSetup, SetupContextWithWallets } from './support.js';
+import { makeDoOffer } from '../tools/e2e-tools.js';
+import { makeQueryClient } from '../tools/query.js';
+
+const test = anyTest as TestFn<SetupContextWithWallets>;
+
+const accounts = ['user1', 'user2'];
+
+test.before(async t => {
+  const { deleteTestKeys, setupTestKeys, ...rest } = await commonSetup(t);
+  // deleteTestKeys().catch();
```

Comment:
In active development, if a test fails with an uncaught exception, the `test.after` block will not run and the key will not be deleted. This will cause the next test run to fail.

The only harm in leaving this is some extra unnecessary work in the happy path (attempting to delete keys that aren't there), so it may be best to include this.

I'm not sure the best pattern has been arrived at yet, so suggestions are welcome. It would be nice if we didn't need to rely on the keyring in the container and could use a new `DirectSecp256k1HdWallet` per run.

Reply:
I've decided to keep this around, as it's harmless. Included a comment about why it's there.